### PR TITLE
Wfp 2425 probation reset work

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

File: app/services/excel/input-scenario-case-data.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const inputCaseData = function (ws, cases, typeTierGroupLength, tiersPerType) {
     for (let i = tiersPerType - 1; i >= 1; i--) {
       casesForThisTier = c.communityCaseNumbers.filter(thisCase => thisCase.tier === i)
       casesForThisTier = casesForThisTier[0]
+      console.log('i=' + i + '/' + tiersPerType + ' case for tier ' + casesForThisTier)
```
Comment (GaReid) [inline app/services/excel/input-scenario-case-data.js:26]: Do we want to leave this log in ?

File: test/unit/services/helpers/test-group-scenario-data.js [hunk 2]
```diff
@@ -7,17 +7,18 @@ let scenarioObjectsArray
 describe('services/helpers/group-scenario-data', function () {
   before(function () {
     scenarioObjectsArray = groupScenarioData(scenarioData)
+    console.log(scenarioObjectsArray)
```
Comment (GaReid) [inline test/unit/services/helpers/test-group-scenario-data.js:10]: As above. Do we want this console log.
This log is in tests so not too important.

Comment (sambailey-moj) [reply test/unit/services/helpers/test-group-scenario-data.js]: fixed thx :)

**Review by GaReid — APPROVED** [review]

**Review by GaReid — APPROVED** [review]
lgtm

**Pull request merged**